websites/management/commands/markdown_cleaning/resource_link_delimiters.py

```diff
@@ -0,0 +1,26 @@
+"""
+WebsiteContentMarkdownCleaner rule to convert
+
+{{< resource_link 8c8f3d8e-da91-817b-219e-e6671b2456a6 "Link Text" >}}
+to
+{{% resource_link 8c8f3d8e-da91-817b-219e-e6671b2456a6 "Link Text" %}}
+"""
+import re
+
+from websites.management.commands.markdown_cleaning.cleanup_rule import (
+    MarkdownCleanupRule,
+)
+from websites.models import WebsiteContent
+
+
+class ResourceLinkDelimitersReplacementRule(MarkdownCleanupRule):
+    """Replacement rule for use with WebsiteContentMarkdownCleaner."""
+
+    regex = r"{{<\sresource_link\s(?P<resource_uuid>[a-z0-9\-]*)\s(?P<link_text>\".*?\")\s>}}"
```

Comment:
**Requested Change:** I don't think this will work for the resource_links that have fragment ids. Also, I'm not 100% sure that all resource link titles have double quotes. I suspect they do? But I'm not sure.

**Suggestion:** We don't really need to parse the arguments into separate uuid/title/fragment for this replacement, so I think something like
```py
regex = r"{{<\sresource_link(?P<the_args>.*?)\s>}}"
```
should work. Plus if we ever needed to do this again in the future, I think the markdown_cleanup rule would be entirely reusable, just with `resource_link` swapped out for a different string.